tasks/workflow/worker.go

```diff
@@ -178,11 +179,8 @@ func (w *worker) handleExecution(t *taskExecution) {
 		}
 	}()
 
-	if taskStatus, err := t.getTaskStatus(); err != nil && taskStatus == tasks.TaskStatusINITIAL {
-		metrics.MeasureSince([]string{"tasks", "wait"}, t.creationDate)
-	}
+	metrics.MeasureSince([]string{"tasks", "wait"}, t.creationDate)
```

Comment:
I'm a bit confused here we call `tasks` anything that relates to tasks executions?

Reply:
Indeed, here we monitor taskExecutions. I fixed this, see the Yorc TaskExecution metrics table in telemetry doc chapter